docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md

```diff
@@ -36,6 +36,12 @@ The following tables list permissions associated with basic and fixed roles.
 The following table has the existing built-in fixed role definitions. Other fixed roles might be added by plugins installed in Grafana.
 The UUID presented here can be used as an identifier for [Terraform provisioning](../rbac-terraform-provisioning).
 
+{{< admonition type="caution" >}}
+These UUIDs might not be available if your instance was created before Grafana v10.2.0.
```

Comment:
```suggestion
These UIDs might not be available if your instance was created before Grafana v10.2.0.
```
the ids are not UUIDs... just unique withint org

Reply:
I think they are unique across all orgs as the role itself is unique across all orgs.